src/components/Explore/TimeSelector.tsx

```diff
@@ -0,0 +1,179 @@
+import { useOnClickOutside } from 'hooks/useOnClickOutside'
+import { useRef } from 'react'
+import React, { useState } from 'react'
+import { Check, ChevronDown, ChevronUp } from 'react-feather'
+import { useModalOpen, useToggleModal } from 'state/application/hooks'
+import { ApplicationModal } from 'state/application/reducer'
+import styled, { css } from 'styled-components/macro'
+
+const TIME_DISPLAYS: { [key: string]: string } = {
+  hour: '1H',
+  day: '1D',
+  week: '1W',
+  month: '1M',
+  year: '1Y',
+}
+
+const enum TimeOption {
+  Hour = 'hour',
+  Day = 'day',
+  Week = 'week',
+  Month = 'month',
+  Year = 'year',
+}
```

Comment:
prob best to just use one of these (pref TimeOptions enum, or exactly the one used in the mock data hooks)